conda-recipe/skll/meta.yaml

```
@@ -39,7 +39,7 @@ requirements:
     - pandas
     - python >=3.6
     - ruamel.yaml
-    - scikit-learn ==0.23.1
+    - scikit-learn ==0.23.2
```

Since it appears `0.23.1` should still work, can we relax this to the following?

```suggestion
    - scikit-learn >=0.23.1,<=0.23.2
```

---

That makes sense @mulhod. Updated.